docs/deprecations.md

```diff
@@ -304,3 +304,13 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 Deprecation code: `BIN_SERVERLESS`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
+
+<a name="CONFIG_VALIDATION_MODE_DEFAULT"><div>&nbsp;</div></a>
```

Comment:
Sorry I've missed it before - let's move that deprecation information to the top of the file, right below the informations on how to disable specific deprecation logs